CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.5.0] - TBD
+### Changed
+- Added additional checks for the number of levels in table and partition paths so that invalid paths don't get scheduled for deletion.
```

Comment:
```suggestion
- Added additional checks for the number of levels in table and partition paths so that invalid paths are not scheduled for deletion.
```